src/sas/qtgui/Plotting/Slicers/WedgeSlicer.py

```diff
@@ -200,14 +200,14 @@ def _post_data(self, new_sector=None, nbins=None):
         new_plot.source = self.data.source
         new_plot.interactive = True
         new_plot.detector = self.data.detector
-        new_plot.plot_role = DataRole.ROLE_ANGULAR_SLICE
         # If the data file does not tell us what the axes are, just assume...
         if self.averager.__name__ == 'SectorPhi':
             # angular plots usually require a linear x scale and better with
             # a linear y scale as well.
             new_plot.xaxis("\\rm{\phi}", "degrees")
             new_plot.xtransform = 'x'
             new_plot.ytransform = 'y'
+            new_plot.plot_role = DataRole.ROLE_ANGULAR_SLICE
         else:
```

Comment:
I would think that the two lines above the added line (`new_plot.ytransform = 'y'`  and `new_plot.xtransform = 'x'`) should at least commented out if not deleted?